### ISSUE TITLE
[Question] I want to know this line need change to using tf.ones function?

### ISSUE THREAD

**Issue by dengc367** [keras-team/keras-io#284]
https://github.com/keras-team/keras-io/blob/e005c9014d19f3dffb0ca983b20eb5a2c3864a3a/guides/customizing_what_happens_in_fit.py#L384

Hello, I want to know this line need use  `tf.ones` function? not the `tf.zeros` labels,  because there need set the `fake` samples to `real`(`1`) label when train the generate model

**Comment by dengc367** [comment]
in the generate training phase,  the label of the fake samples need set to `1`, 
like the 'valid' value in the 
https://github.com/eriklindernoren/Keras-GAN/blob/636ec0533df1d1ba2bfe4ec9ae7aa66bd7ee2177/dcgan/dcgan.py#L116-L143

**Comment by dengc367** [comment]
Hello @fchollet ,please help to check that what I thought above is true?
I have read the keras GAN end-to-end training tutorial from the [tensorflow guide docs](https://tensorflow.google.cn/guide/keras/customizing_what_happens_in_fit/)
 I look forward to the reply from keras team. Thanks.

**Comment by dengc367** [comment]
I train the code using `tf.ones`, the loss is divergence; but using `tf.zeros`, the loss is converge. so wield.

> https://github.com/keras-team/keras-io/blob/e005c9014d19f3dffb0ca983b20eb5a2c3864a3a/guides/customizing_what_happens_in_fit.py#L384
> 
> Hello, I want to know this line need use `tf.ones` function? not the `tf.zeros` labels, because there need set the `fake` samples to `real`(`1`) label when train the generate model

**Comment by dengc367** [comment]
I got the answer now, thanks. [Question about 'label' in training of GAN](https://github.com/fchollet/deep-learning-with-python-notebooks/issues/9#issuecomment-334048531)
